src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
+    double area();

```

Comment:
method names should be verbs
```suggestion
    double getArea();

```